templates/home/layouts/home.html

```diff
@@ -37,15 +37,15 @@
 		<div class="row">
 			{%- unless page.nomenu -%}
 				{%- if page.auth -%}
-					<div class="col-sm-9">
+					<div class="col-md-8">
 				{%- else -%}
 					<div class="col-sm-12">
 				{%- endif -%}
 				{% include gcweb-menu/sitemenu.html %}
 			{%- endunless -%}
 		</div>
 		{%- if page.auth -%}
-			<div class="col-sm-3">
+			<div class="col-xs-5 col-xs-offset-7 col-md-offset-0 col-md-4">
```

Comment:
These changes will be included as part of #1977 as well, so this is just a note for ourselves to take this into considerations. Documentation for this change on the home page however is in the other PR while this current PR is more pressing. One other thing to consider though is the fact that we simply forgot this template when we first implemented the Contextual sign in button.

CC: @Garneauma @Ricokola